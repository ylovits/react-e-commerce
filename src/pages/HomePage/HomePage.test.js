import React from "react";
import { shallow } from "enzyme";
import Homepage from "./Homepage";

it("<Homepage /> rendered", () => {
  expect(shallow(<Homepage />)).toMatchSnapshot();
});
