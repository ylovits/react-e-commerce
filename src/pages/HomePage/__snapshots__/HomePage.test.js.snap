// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Homepage /> rendered 1`] = `ShallowWrapper {}`;
